virtwho/manager/subscriptionmanager/subscriptionmanager.py

```diff
@@ -179,7 +179,8 @@ def hypervisorCheckIn(self, report, options=None):
 
         is_async = self._is_rhsm_server_async(report, connection)
         serialized_mapping = self._hypervisor_mapping(report, is_async, connection)
-        self.logger.debug("Host-to-guest mapping: %s", json.dumps(serialized_mapping, indent=4))
+        self.logger.debug("Host-to-guest mapping being sent to '%s': %s", report.config.owner,
```

Comment:
@cnsnyder not sure if this works, and I dont think all the counts are necessary. I think we can fix this in a single line change:

\-                self.logger.info('Hosts-to-guests mapping for config "%s": %d hypervisors and %d guests found',  report.config.name, hypervisor_count, guest_count)
\+                self.logger.info('Hosts-to-guests mapping for config "%s": %d hypervisors and %d guests found for %s',  report.config.name, hypervisor_count, guest_count, report.config.owner)


thoughts?